app/preview/preview.html

```diff
@@ -1,10 +1,6 @@
 <div class="preview-frame"><iframe ng-src="{{ url }}" name="preview-frame"></iframe></div>
-<table class='table table-hover table-condensed'>
-  <tr ng-repeat="file in files.files" ng-class="{active: file.id === id}">
-    <td>
-      <a ng-href="#{{ $routeSegment.getSegmentUrl('preview', {id: file.id}) }}">{{ file.title }}</a>
-      <a ng-href="fixtures/content/{{ file.title }}" download="filename">(Download)</a>
-    </td>
-  </tr>
-</table>
-<p ng-if='files.files.length === 0'>No files selected</p>
+<p ng-if='file'>
+  <a ng-href="#{{ $routeSegment.getSegmentUrl('preview', {id: file.id}) }}">{{ file.title }}</a>
```

Comment:
I don't think we need a link to the preview pane, since the only possible condition for this link to be displayed is when it's already at this pane?